requester/requester_test.go

```diff
@@ -133,3 +135,45 @@ func TestBody(t *testing.T) {
 		t.Errorf("Expected to work 10 times, found %v", count)
 	}
 }
+
+func TestOutput(t *testing.T) {
+	var count int64
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		atomic.AddInt64(&count, int64(1))
+	}
+	server := httptest.NewServer(http.HandlerFunc(handler))
+	defer server.Close()
+
+	req, _ := http.NewRequest("GET", server.URL, nil)
+	w := &Work{
+		Request: req,
+		N:       20,
+		C:       2,
+		Output:  "csv",
+	}
+	stdOut := readStdout(w.Run)
+	if count != 20 {
+		t.Errorf("Expected to work 20 times, found %v", count)
+	}
+	scanner := bufio.NewScanner(bufio.NewReader(stdOut))
+	var csvData []string
+	for scanner.Scan() {
+		csvData = append(csvData, scanner.Text())
+	}
+	if csvData[0] != "response-time" {
+		t.Error("Expected csv response-time header")
+	}
+	if len(csvData) != 21 {
```

Comment:
if got, want := len(csvData), 21; got != want {
t.Errorf("number of lines of csv output = %v; want %v", got, want)
}